### ISSUE TITLE
Large Mordor Map crashing when attacking neutral territories

### ISSUE THREAD

**Issue by justbleh** [triplea-game/triplea#2594]
### My Operating System:
windows 64 bit

[large morder hard good.zip](https://github.com/triplea-game/triplea/files/1464039/large.morder.hard.good.zip)

### TripleA version:

1.9.0.0.7378
### Map:
Large Mordor
### Can you describe how to trigger the error? (eg: what sequence of actions will recreate it?)
I saved it right at the error, initiating the battle
### Do you have the exact error text? Please copy/paste if so
```
triplea.engine.version.bin:1.9
Loading map: large_middle_earth, from: C:\Users\johnw_000\triplea\downloadedMaps\large_middle_earth-master.zip
Loading resources from the following paths: [C:\Users\johnw_000\triplea\downloadedMaps\large_middle_earth-master.zip, C:\Program Files\TripleA\assets]
Loading map: large_middle_earth, from: C:\Users\johnw_000\triplea\downloadedMaps\large_middle_earth-master.zip
Loading resources from the following paths: [C:\Users\johnw_000\triplea\downloadedMaps\large_middle_earth-master.zip, C:\Program Files\TripleA\assets]
java.lang.reflect.UndeclaredThrowableException
	at com.sun.proxy.$Proxy13.selectCasualties(Unknown Source)
	at games.strategy.triplea.delegate.BattleCalculator.selectCasualties(BattleCalculator.java:514)
	at games.strategy.triplea.delegate.Fire.selectCasualties(Fire.java:161)
	at games.strategy.triplea.delegate.Fire.access$100(Fire.java:21)
	at games.strategy.triplea.delegate.Fire$2.execute(Fire.java:221)
	at games.strategy.triplea.delegate.ExecutionStack.execute(ExecutionStack.java:41)
	at games.strategy.triplea.delegate.MustFightBattle.fight(MustFightBattle.java:380)
	at games.strategy.triplea.delegate.BattleTracker.fightBattleIfOnlyOne(BattleTracker.java:1176)
	at games.strategy.triplea.delegate.BattleDelegate.start(BattleDelegate.java:97)
	at games.strategy.engine.framework.ServerGame.startStep(ServerGame.java:506)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:401)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:253)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$169(AbstractLauncher.java:51)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.delegate.DelegateExecutionManager.lambda$createOutboundImplementation$113(DelegateExecutionManager.java:116)
	... 15 more
Caused by: games.strategy.engine.message.RemoteNotFoundException: Not found:games.strategy.engine.framework.ServerGame.PLAYER_REMOTE.Neutral
	at games.strategy.engine.message.UnifiedMessengerHub.messageReceived(UnifiedMessengerHub.java:89)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.send(UnifiedMessenger.java:224)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.invokeAndWaitRemote(UnifiedMessenger.java:122)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.invokeAndWait(UnifiedMessenger.java:99)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:51)
	at com.sun.proxy.$Proxy13.selectCasualties(Unknown Source)
	... 20 more
Caused by: java.lang.IllegalStateException: remote not found
	at java.lang.Thread.getStackTrace(Unknown Source)
	at games.strategy.engine.message.MessengerException.fillInInvokerStackTrace(MessengerException.java:19)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:55)
	... 21 more
```

### Instead of this error, what should have happened?
neutral territories select kills.

**Comment by ssoloff** [comment]
Confirmed that I can reproduce this exception with the attached save game in 1.9.0.0.7496.

If I revert #2557 locally, the exception is not thrown and the battle proceeds as expected.

**Comment by ssoloff** [comment]
@justbleh This should be fixed in pre-release 1.9.0.0.7509 or later.  You can download the latest pre-release [here](https://github.com/triplea-game/triplea/releases).  Please re-open this issue if you still experience the error.

**Comment by justbleh** [comment]
Perfect, my saved game even runs with the new update. Thanks for you speedy reply and help! I appreciate your thoroughness and proficiency.
